app/core/models/ocelot/package.scala

```diff
@@ -48,7 +48,7 @@ package object ocelot {
   def labelReferences(str: String): List[String] = plSingleGroupCaptures(labelRefRegex, str)
   def labelReference(str: String): Option[String] = plSingleGroupCaptures(labelRefRegex, str).headOption
   def asTextString(value: String): Option[String] = value.trim.headOption.fold[Option[String]](None)(_ => Some(value.trim))
-  def asCurrency(value: String): Option[BigDecimal] = inputCurrencyRegex.findFirstIn(value.filterNot(c => c==' '))
+  def asDecimal(value: String): Option[BigDecimal] = inputCurrencyRegex.findFirstIn(value.filterNot(c => c==' '))
```

Comment:
The sequence of chars, ignoredCurrencyChars, includes a space character.

Reply:
the match is done ignoring spaces, but the value determination uses the ignoredCurrencyChars list